contributors/design-proposals/cri-dockershim-checkpoint.md

```diff
@@ -0,0 +1,91 @@
+# CRI: Dockershim PodSandbox Checkpoint 
+
+## Umbrella Issue
+[#34672](https://github.com/kubernetes/kubernetes/issues/34672)
+
+## Background
+[Container Runtime Interface (CRI)](../devel/container-runtime-interface.md) is an ongoing project to allow container runtimes to integrate with kubernetes via a newly-defined API. [Dockershim](https://github.com/kubernetes/kubernetes/blob/release-1.5/pkg/kubelet/dockershim) is the Docker CRI implementation. This proposal aims to introduce checkpoint mechanism in dockershim.
```

Comment:
nit (and probably personal preference): limiting each line to 80 chars by inserting newlines makes review easier.